fresh/tests/features.t

```diff
@@ -330,7 +330,6 @@ np:.p.import[`numpy]
 /For the testing of quantiles the 'y' argument must be in the range [0;1] by definition
 .ml.fresh.feat.quantile[xj;0.5] ~ quantile_py[xj;0.5]
 .ml.fresh.feat.quantile[xf;0.5] ~ quantile_py[xf;0.5]
-.ml.fresh.feat.quantile[xb;0.5] ~ quantile_py[xb;0.5]
```

Comment:
Why is this being removed?

Reply:
Python function does not support binary computations anymore